x-pack/plugins/security_solution/public/management/components/console/components/console_state/state_update_handlers/handle_execute_command.tsx

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+/* eslint-disable complexity */
```

Comment:
is this necessary?

I'm thinking its your IDE that is complaining not really how the rules are setup for eslint in kibana. can you confirm

Reply:
yeah not necessary. Will remove it

Reply:
99e13666cb030e03427e48b3bf935b09b293207d